var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -32,6 +34,7 @@ class Subversion(AutotoolsPackage):
 
     variant('serf', default=True,  description='Serf HTTP client library')
     variant('perl', default=False, description='Build with Perl bindings')
+    variant('gettext', default=True, description='Build with gettext functionality')
```

Comment:
I think in other packages `nls` (build with native language support) is used as variant name

Reply:
I usually just go with the name of the configure/cmake flag, which in this case is `nls`.

Reply:
Fixed!!